openflow13/match.go

```diff
@@ -86,7 +87,11 @@ func (m *Match) UnmarshalBinary(data []byte) error {
 
 	for n < int(m.Length) {
 		field := new(MatchField)
-		field.UnmarshalBinary(data[n:])
+		err := field.UnmarshalBinary(data[n:])
```

Comment:
This error check should be inline in the `if` statement because `err` is never used again